File: Python/Hardware.py
from Product import Product

class Hardware(Product):

    def __init__(self, id, p, b, m):
        super().__init__(id, p)
        self.brand = b
        self.model = m

    def setBrand(self, b):
        self.brand = b
    def getBrand(self):
        return self.brand

    def setModel(self, m):
        self.model = m
    def getModel(self):
        return self.model

File: PHP/Hardware.php
<?php

include "Product.php";

class Hardware extends Product{
    private $brand = "VGen";
    private $model;

    public function __construct($id, $p, $b, $m){
        $this->brand = $b;
        $this->model = $m;
        $this->setIDProduct($id);
        $this->price = $p;
    }

    public function setBrand($b){
        $this->brand = $b;
    }

    public function getBrand(){
        return $this->brand;
    }

    public function setModel($m){
        $this->model = $m;
    }

    public function getModel(){
        return $this->model;
    }
}

?>